### PR TITLE
fix: add badbatunde

### LINKED ISSUE #1300: HBO
## Profile Screenshot
hbo

### PR DESCRIPTION
**Pull request by Badbatunde**
closes #1300

## Checklist for profile addition

- [x] Name the PR `fix: add badbatunde`.
- [x] You've added file named `badbatunde.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

### CODE REVIEW

**Review by MrKrishnaAgarwal — APPROVED** [review]
Nice Profile 👌🏻

File: profiles/badbatunde.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/105541302/184447490-f249bc52-23fa-4249-abed-8d8d2b97392b.png",
+  "issueId": 1300,
+  "name": "HBO",
```
Comment (MrKrishnaAgarwal) [inline profiles/badbatunde.json:4]: but consider entering your full name.
```suggestion
  "name": "HBO",
```

Comment (Badbatunde) [reply profiles/badbatunde.json]: Thanks, I have made the modification.

**Review by MrKrishnaAgarwal — APPROVED** [review]
Great 👍🏻
You are good to go!

Follow me on GitHub :)

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Badbatunde** [comment]
Thank you 😊

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge:tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Pull request merged**